modules/rosa-list-objects.adoc

```diff
@@ -0,0 +1,457 @@
+<id> | <name>// Module included in the following assemblies:
```

Comment:
Typo here of ` <id> | <name>`

Reply:
I saw a typo in the examples for `name`, which I just fixed. but I don't see a typo for <id> | <name>. If I am misunderstanding, please let me know.